output.c

```diff
@@ -42,6 +43,25 @@
 #include "xwayland.h"
 #endif
 
+static void
+update_output_manager_config(struct cg_server *server)
+{
+	struct wlr_output_configuration_v1 *config = wlr_output_configuration_v1_create();
+
+	struct cg_output *output;
+	wl_list_for_each (output, &server->outputs, link) {
+		struct wlr_output_configuration_head_v1 *config_head =
+			wlr_output_configuration_head_v1_create(config, output->wlr_output);
+		struct wlr_box *box = wlr_output_layout_get_box(server->output_layout, output->wlr_output);
+		if (box) {
+			config_head->state.x = box->x;
+			config_head->state.y = box->y;
+		}
```

Comment:
Should this also set `config_head->state.enabled` and `config_head->state.mode`?

Reply:
Can you also have this set `state.scale` and `scale.transform`?

Reply:
Amended this in your branch :)